software/open-quartermaster.yml

```diff
@@ -0,0 +1,19 @@
+name: "Open QuarterMaster"
+website_url: "https://openquartermaster.com/"
+source_code_url: "https://github.com/Epic-Breakfast-Productions/OpenQuarterMaster"
+description: "A powerful inventory management system, designed to be flexible and scalable."
+licenses:
+  - GPL-3.0
+platforms:
+  - Java
+  - Python
+  - PHP
+  - Deb
+  - Docker
+tags:
+  - Inventory Management
+  - Analytics
+  - Groupware
+  - Manufacturing
+  - Internet of Things (IoT)
+depends_3rdparty: false
```

Comment:
```suggestion
```
This is a redundant line too as its already defaulted to false 👍🏻

Reply:
It was in the example yaml :) But fine either way.

Reply:
yeah, that's fair. You would only really need to include it if it depends on third party stuff. Might need to fix